tests/testutil/service.go

```diff
@@ -209,7 +284,7 @@ func (ts *TestService) GetImage() string {
 
 // GetRunningPods returns all running pods, separated by new lines, for this service's app
```

Comment:
Believe this comment is no longer correct. The name isn't very descriptive either, I would assume it returns pod objects.